docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -0,0 +1,72 @@
+- Less boilerplate so you can focus on the important parts of the app
+
+## Market overview
+
+After reviewing multiple approaches and tools, we noticed that there are a few recurring core problem areas that needed to be addressed:
+
+### The challenge of codebase fragmentation and duplication
+
+Frequently, developers invest valuable time in hunting down particular versions of business logic and then maintaining them at multiple places. When working with external tools, code duplication can often arise as consequences of design decisions. Not having a unified reference point within a codebase compromises its scalability, ease of maintenance and debugging.
+
+### Loss of control with external tools
+
+When working with external tools, developers often forfeit their autonomy and are constrained by the choices offered by the tool. Developers typically derive a sense of ownership and responsibility when they oversee the entire process, encompassing integrations, deployment, and version control. However, when a black-box element intervenes, control often shifts away.
+
+### Trade-off between speed and customization
+
+Typically, depending on factors such as project size, time constraints, available resources, and scope, developers are compelled to strike a balance between speed and customization. Every business wants features to be delivered at speed, but being able to customize later keeps the project open to come what may. While both aspects are significant, we believe that an architecturally well-designed tool empowers you to attain both objectives simultaneously.
+
+### Trust and Safety
+
+Internal tools have access to highly sensitive data, data that every organization solemnly pledges to safeguard. Despite the stringent security measures in place, data breaches remain a prevalent occurrence. Organizations rigorously engage in due diligence and compliance assessments before placing trust in a vendor. The security of the underlying data remains an exceptionally vital aspect that cannot be underestimated.
+
+## Toolpad
+
+We think addressing the above mentioned problems can lead to a better developer experience. Toolpad solves them in the following ways:
+
+### Front-end UI
+
+Toolpad works as a drag-and-drop UI builder and uses MUI's own Material UI components. Material UI has almost every component required for internal use cases. Currently, Toolpad supports 16 of them and work on supporting more is continuously ongoing.
+
+For additional use-cases, Toolpad supports adding any number of custom components. You can create your own by writing React code. The drag-and-drop builder helps you assemble a quick UI and gives you a head start for the next steps. It ensures **speed and customization**.
+
+### Connecting to data
+
+Toolpad supports connecting to data in two ways:
+
+- REST APIs can be easily linked to Toolpad through _HTTP queries_. This is the most common way of getting data to Toolpad.
```

Comment:
numbered list, active voice ("do this" instead of "this can be done")

```suggestion
1. Link REST APIs to Toolpad through HTTP queries—this is the most common way of getting data to Toolpad.
```